bundle/bundle.go

```diff
@@ -48,6 +48,12 @@ type Bundle struct {
 	// Exclusively use this field for filesystem operations.
 	SyncRoot vfs.Path
 
+	// Path to the root of git worktree containing the bundle.
+	// This is the same as git repository root if worktrees are not used,
+	// otherwise it's a descedant of git repository root.
```

Comment:
> it's a descedant of git repository root

Not guaranteed. Worktrees can live anywhere.

Reply:
Good to know, updated.